ark/model/model_node.cpp

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "model_node.hpp"
+
+namespace ark {}  // namespace ark
```

Comment:
Why is empty?

Reply:
Removed this file in [1fbf38b](https://github.com/microsoft/ark/pull/205/commits/1fbf38b4dd839f97199ade30a90959c5145f1a86).